lib/rules/no-else-return.js

```diff
@@ -12,6 +12,35 @@
 const astUtils = require("../util/ast-utils");
 const FixTracker = require("../util/fix-tracker");
 
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+const BLOCK_SCOPED_DECLARATION = /^(?:VariableDeclarator|ClassDeclaration)$/;
+
+/**
+ *
+ * @param {*} scope  current scope
+ * @param {Token} startToken s
+ * @param {Token} endToken s
+ * @returns {boolean} s
+ */
+function blockScopeDeclaration(scope, startToken, endToken) {
```

Comment:
I'm a little confused by this function name. What are we checking here? It looks like we might be trying to check if an `else` block has any block scope declarations?

I would suggest prefixing the function name with "is" or "has", if it is returning a boolean.

Reply:
I have renamed it. 
Yes, I'm checking here if `else` block has any scope declarations,